src/lib/types.ts

```diff
@@ -797,8 +797,7 @@ export interface GoTrueMFAApi {
    * Returns the list of MFA factors enabled for this user. For most use cases
    * you should consider using {@link
    * GoTrueMFAApi#getAuthenticatorAssuranceLevel}. This uses a cached version
-   * of the factors and avoids incurring a network call. If you need to update
-   * this list, call {@link GoTrueClient#getUser} first.
+   * of the factors and avoids incurring a network call.
```

Comment:
hmm just wondering why this change is necessary? @dshukertjr

Reply:
@kangmingtay Because with [this](https://github.com/supabase/gotrue-js/pull/510) PR, `getUser()` is always called within `listFactors()`, therefore the developer does not have to call `getUser()` first like it is suggested in the comments to update the list.